docs/content/getting-started/setup-openfga.mdx

```diff
@@ -163,18 +163,29 @@ In a terminal, navigate to that directory and run:
 ```shell
 docker-compose up
 ```
+
+This will start the Postgres database, run `openfga migrate` to configure the database and finally start the OpenFGA server.
```

Comment:
This should be outside the tabs - it applies to all

Reply:
I know, but if you look at the Docker instructions, i think it is pretty readable

<img width="817" alt="image" src="https://user-images.githubusercontent.com/5374887/222858166-bc27bd77-5233-4057-9a81-7767e366e6bb.png">

Reply:
Oh, I meant how it was before: Rremove the sentence you added here https://github.com/openfga/openfga.dev/pull/379/files#diff-f3dd1b42a6d15002efd082f00d258df0591a356412666d0ec66ddad9ceeb0b52R100

And move this out of the tab, you get the same output without having to duplicate the content.